charts/lws/templates/webhook/service.yaml

```diff
@@ -12,4 +12,5 @@ spec:
       protocol: TCP
       targetPort: 9443
   selector:
+    {{- include "lws.selectorLabels" . | nindent 4 }}
```

Comment:
Have you ever tried with this? Just to confirm this works as expected.

Reply:
Yes, please see 

```
(base) ➜  lws git:(fix-chart) helm install llmaz ./charts/lws
NAME: llmaz
LAST DEPLOYED: Wed May 21 15:24:51 2025
NAMESPACE: llmaz-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
(base) ➜  lws git:(fix-chart) kubectl get svc -ojsonpath="{.items[0].spec.selector}"
{"app.kubernetes.io/instance":"llmaz","app.kubernetes.io/name":"lws","control-plane":"controller-manager"}%                                                                                                 (base) ➜  lws git:(fix-chart) kubectl get pod -ojsonpath="{.items[0].metadata.labels}"
{"app.kubernetes.io/instance":"llmaz","app.kubernetes.io/name":"lws","control-plane":"controller-manager","pod-template-hash":"5b7f6b4d4f"}%
```